spago.yaml

```
@@ -0,0 +1,34 @@
+package:
+  name: node-streams
+  publish:
+    license: MIT
+    version: 9.0.0
+    location:
+      githubOwner: purescript-node
+      githubRepo: purescript-node-streams
+  dependencies:
+    - effect: ">=4.0.0 <5.0.0"
+    - either: ">=6.0.0 <7.0.0"
+    - exceptions: ">=6.0.0 <7.0.0"
+    - node-buffer: ">=9.0.0 <10.0.0"
+    - nullable: ">=6.0.0 <7.0.0"
+    - prelude: ">=6.0.0 <7.0.0"
+    - node-event-emitter: ">=3.0.0 <4.0.0"
+    - aff: ">=8.0.0 <9.0.0"
+  test:
+    main: Test.Main
+    dependencies:
+      - assert
+      - console
+      - partial
+      - spec
+
+workspace:
+  extraPackages:
+    assert: 6.0.0
+    console: 6.0.0
+    partial: 4.0.0
+    spec: 8.0.0
+    spec-node: 0.0.2
+  packageSet:
+    registry: 55.1.0
```

This is currently building off a package set, but for libraries I'd recommend using a solver build, so that you can exercise the package bounds specified in the dependencies:
```suggestion
workspace: {}
```

---

Thank you for reviewing @f-f